mylyn.github/pom.xml

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (C) 2011, Chris Aniszczyk <caniszczyk@gmail.com>
+   Copyright (C) 2011, Matthias Sohn <matthias.sohn@sap.com>
+
+   All rights reserved. This program and the accompanying materials
+   are made available under the terms of the Eclipse Public License 2.0
+   which accompanies this distribution, and is available at
+   https://www.eclipse.org/legal/epl-2.0/
+
+   SPDX-License-Identifier: EPL-2.0
+-->
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.mylyn.github</groupId>
+  <artifactId>github-parent</artifactId>
+  <version>6.2.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Eclipse EGit Mylyn GitHub Connector Parent</name>
+
+  <mailingLists>
+    <mailingList>
+      <name>egit-dev Mailing List</name>
```

Comment:
Should we switch mail list?

Reply:
Yes

Reply:
I also changed this and include the GitHub Connector in the maven build and the updatesite.